libs/wingsdk/src/target-tf-gcp/platform.ts

```diff
@@ -0,0 +1,16 @@
+import { App } from "./app";
+import { IPlatform } from "../platform";
+
+/**
+ * GCP Terraform Platform
+ */
+export class TFGCPPlatform implements IPlatform {
+  /** Platform model */
+  public readonly model = "tf-gcp";
+
+  public newApp?(appProps: any): any {
+    return new App(appProps);
+  }
+}
+
+export const Platform = TFGCPPlatform;
```

Comment:
I think this can be
```suggestion
export class Platform implements IPlatform {
  /** Platform model */
  public readonly model = "tf-gcp";

  public newApp?(appProps: any): any {
    return new App(appProps);
  }
}
```